packages/synthetix-main/contracts/storage/Distribution.sol

```diff
@@ -4,42 +4,140 @@ pragma solidity ^0.8.0;
 import "@synthetixio/core-contracts/contracts/utils/MathUtil.sol";
 
 import "./DistributionActor.sol";
+import "../errors/ParameterError.sol";
 
+/**
+ * @title Tracks value distributed amongst a set of actors using shares. A share represents a unit in which the total value is distributed, and thus the granilarity of the distribution.
```

Comment:
"tracks value distributed" usually (at least in english) repeating terms which are used in the title of something is not helpful in a description. maybe it would be better to say somethign like "In a pattern typical to the longstanding `StakingRewards` contract, divides a quanitity of `value` between a set of actors proportional to the number of shares they have"

Reply:
Oh I really like the last phrase! I would prefer to not refer to any v2x contracts, or even assume previous knowledge of that. Better if we can explain everything with no previous context.

Reply:
Updated